packages/docusaurus-theme-translations/locales/fr/theme-common.json

```
@@ -32,6 +32,9 @@
   "theme.common.editThisPage": "Éditer cette page",
   "theme.common.headingLinkTitle": "Lien direct vers le titre",
   "theme.common.skipToMainContent": "Aller au contenu principal",
+  "theme.created.atDate": " on {date}",
```

```suggestion
  "theme.created.atDate": " le {date}",
```

Proposal for the good translation into French from now on